bot/src/infrastructure/monitoring/MetricsCollector.ts

```
@@ -0,0 +1,390 @@
+      lines.push(`${name}${labelStr} ${value}`);
+    }
+    
+    // Histograms
+    for (const [key] of this.histograms) {
+      const { name, labels } = this.parseMetricKey(key);
+      const stats = this.getHistogramStats(name, labels);
+      if (stats) {
+        const labelStr = labels ? `{${Object.entries(labels).map(([k, v]) => `${k}="${v}"`).join(',')}}` : '';
+        lines.push(`${name}_count${labelStr} ${stats.count}`);
+        lines.push(`${name}_sum${labelStr} ${stats.sum}`);
+        lines.push(`${name}_bucket{le="95"${labels ? ',' + Object.entries(labels).map(([k, v]) => `${k}="${v}"`).join(',') : ''}} ${stats.p95}`);
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)